### PR TITLE
Testing to do list part 2

### PR DESCRIPTION
**Pull request by ismayito**
###Summary:
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------
we have tested the following
- Clear complete all functionality
- edit task functionality
- Select item functionality

### CODE REVIEW

File: src/modules/deleteItem.js [hunk 1]
```diff
@@ -3,12 +3,19 @@ import listStore from './listStore.js';
...
+    });
+  }
+  // const returnedItems = retrieveFromStore().filter((item) => item.index !== id);
   // update the indices of the remaining items
-  returnedItems.forEach((item, index) => {
-    item.index = index;
-  });
+  // returnedItems.forEach((item, index) => {
+  //   item.index = index;
+  // });
```
Comment (anagudelogu) [inline src/modules/deleteItem.js:18]: - [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧

File: src/modules/displayList.js [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import retrieveFromStore from './retrieveFromStore.js';
 import selectItem from './selectItem.js';
 import deleteItem from './deleteItem.js';
-import listStore from './listStore.js';
-
 // eslint-disable-next-line import/no-cycle
```
Comment (anagudelogu) [inline src/modules/displayList.js:4]: - [ ] Please avoid disabling linters as they are very useful to keep a consistent code format and ensure the use of best practices. Instead, please consider solving the errors! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @ismayito @Goldinium, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @ismayito 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️commented code removed
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**